ntbs-service/Services/AzureAdDirectoryService.cs

```diff
@@ -240,9 +241,9 @@ private bool IsGroupNameValidAndUnique(string groupName, IEnumerable<string> gro
                 GivenName = graphUser.GivenName,
                 FamilyName = graphUser.Surname,
                 DisplayName = displayName,
-                IsActive = graphUser.AccountEnabled.HasValue && graphUser.AccountEnabled.Value,
-                AdGroups = string.Join(",", groupNames),
-                IsCaseManager = tbServicesMatchingGroups.Any()
+                IsActive = accountEnabled,
+                AdGroups = accountEnabled ? string.Join(",", groupNames) : null,
+                IsCaseManager = accountEnabled && tbServicesMatchingGroups.Any()
```

Comment:
Technically, we don't need the `accountEnabled &&` here, since when `accountEnabled` is false we know `tbServicesMatchingGroups` will be empty anyway. However, it doesn't do any harm.